### PR TITLE
Simplify presentation of BP readings in RRD hypertension PDF. Adds caveats to PDF.

### PR DESCRIPTION
**Pull request by allthesignals**
## Description of change
Simplifies the presentation of the BP readings. Makes changes to some of the copy of the hypertension PDF, specifically the about section. Adds some spacing fixes.

## Original issue(s)
https://www.notion.so/PDF-Add-caveats-to-PDF-Lighthouse-Data-cf0920ed891b4738a41303f5e30a035a?d=9541f3d6fd8540419793b535350869cf

### CODE REVIEW

**Review by yoomlam — DISMISSED** [review]
🎉

File: app/services/rapid_ready_for_decision/hypertension_pdf_generator.rb [hunk 1]
```diff
@@ -44,7 +44,8 @@ def add_intro
         "<font size='11'>Hypertension Rapid Ready for Decision | Claim for Increase</font>\n",
         "<font size='22'>VHA Hypertension Data Summary for</font>",
         "<font size='22'>#{patient_name}</font>\n",
-        "<font size='10'><i>Generated automatically on #{generated_at}<i>\n"
+        "<font size='10'><i>Generated automatically on #{generated_at}<i>\n",
```
Comment (thilton-oddball) [inline app/services/rapid_ready_for_decision/hypertension_pdf_generator.rb:47]: Does this line need a closing `</font>` tag?

File: app/services/rapid_ready_for_decision/hypertension_pdf_generator.rb [hunk 2]
```diff
@@ -78,7 +79,10 @@ def add_blood_pressure_intro
 
     def add_blood_pressure_list
       @blood_pressure_data.each do |bp|
-        @pdf.text "<b>Blood pressure: #{bp[:systolic]['value']}/#{bp[:diastolic]['value']} #{bp[:systolic]['unit']}",
+        systolic = bp[:systolic]['value'].round
+        diastolic = bp[:diastolic]['value'].round
+
+        @pdf.text "<b>Blood pressure: #{systolic}/#{diastolic}",
```
Comment (thilton-oddball) [inline app/services/rapid_ready_for_decision/hypertension_pdf_generator.rb:85]: Does this line need a closing `</b>` tag?

File: app/services/rapid_ready_for_decision/hypertension_pdf_generator.rb (hunk 1 above)
Comment (allthesignals) [reply app/services/rapid_ready_for_decision/hypertension_pdf_generator.rb:47]: YES, thank you. I would love to find a way to use some kind of template for this stuff.

**Comment by allthesignals** [comment]
Thanks @thilton-oddball I went through and fixed any other unclosed tags.

**Review by LindseySaari — APPROVED** [review]

**Pull request merged**